### ISSUE TITLE
[Question] Memory requiremsnts for ORPOTrainer

### ISSUE THREAD

**Issue by snassimr** [xfactlab/orpo#13]
I found this code to run trl's ORPOTrainer [link](https://github.com/xfactlab/orpo/blob/main/trl/test_orpo_trainer_demo.py)
What GPU memory I need to run phi-2 ? QLora isn't available as I understand.

**Comment by nlee-208** [comment]
Hi @snassimr,
It is hard to say as training is dependent on diverse aspects such as batch size, input & output length, availability of distributed training, etc. I'd recommend starting on truncating the input lengths with smaller batch sizes and correspondingly higher gradient accumulation. Maybe consider going through a related [documentation](https://huggingface.co/docs/transformers/en/perf_train_gpu_one) on Transformers
